### ISSUE TITLE
Nether not generating

### ISSUE THREAD

**Issue by AstroxV** [Drakma/VacantSky#31]
In my singleplayer world i noticed there were many chunks not loaded in the nether, they were big holes to the void. So i decided to regenerate the nether by deleting the DIM-1 folder (which normally just regenerates the nether when you re-enter it). But the moment i entered it, there was nothing there. No nether portal, no nothing. I flew around a bit and noticed nether fortresses do generate.
I'm starting to wonder if the Nether is supposed to be a normal nether world, or is it supposed to be a void nether?

I'm trying to see if something is wrong with the pack, or just my world or maybe my computer. Will report back if i find more.

**Comment by Drakma** [comment]
It's supposed to be a void world

On Dec 18, 2016, at 6:01 PM, AstroxV <notifications@github.com<mailto:notifications@github.com>> wrote:

In my singleplayer world i noticed there were many chunks not loaded in the nether, they were big holes to the void. So i decided to regenerate the nether by deleting the DIM-1 folder (which normally just regenerates the nether when you re-enter it). But the moment i entered it, there was nothing there. No nether portal, no nothing. I flew around a bit and noticed nether fortresses do generate.
I'm starting to wonder if the Nether is supposed to be a normal nether world, or is it supposed to be a void nether?

I'm trying to see if something is wrong with the pack, or just my world or maybe my computer. Will report back if i find more.

**Comment by AstroxV** [comment]
Then something went wrong with the creation of my world when i started playing...
I tried all kind of things to fix it.
Created a new world and copied the dimension over, then recreate my nether portal in my existing world. But each time i go through the map is empty, no portal, no tiny platform to stand on, nothing...
I am assuming i'm missing some files to move over, but i cant figure out which ones. Any idea how to fix this?

**Comment by Drakma** [comment]
Is this server or single player?
On Dec 19, 2016, at 5:59 AM, AstroxV <notifications@github.com<mailto:notifications@github.com>> wrote:

Then something went wrong with the creation of my world when i started playing...
I tried all kind of things to fix it.
Created a new world and copied the dimension over, then recreate my nether portal in my existing world. But each time i go through the map is empty, no portal, no tiny platform to stand on, nothing...
I am assuming i'm missing some files to move over, but i cant figure out which ones. Any idea how to fix this?

**Comment by AstroxV** [comment]
Its singleplayer. I gave up with it for now. I guess it made it a bit easier for me for some materials, but I don't really need the nether anymore I think. I tried recreating new worlds with any voidmap template and all have the same nether, a void one with only the portal. I don't want to mess up my overworld, so I don't bother with it anymore. Because its the first time I'm playing a skyblock map I had no idea the nether had to be void too.

**Comment by Drakma** [comment]
That sounds right.  The nether should be a void world with just a portal.  There should be fortresses too.
On Dec 19, 2016, at 3:32 PM, AstroxV <notifications@github.com<mailto:notifications@github.com>> wrote:

Its singleplayer. I gave up with it for now. I guess it made it a bit easier for me for some materials, but I don't really need the nether anymore I think. I tried recreating new worlds with any voidmap template and all have the same nether, a void one with only the portal. I don't want to mess up my overworld, so I don't bother with it anymore. Because its the first time I'm playing a skyblock map I had no idea the nether had to be void too.
